Zilon.Core/CDT.LAST.MonoGameClient/Resources/UiResources.Designer.cs

```diff
@@ -60,414 +45,285 @@ internal class UiResources {
+                return ResourceManager.GetString("SlotBody", resourceCulture);
             }
         }
         
-        /// <summary>
-        ///   Looks up a localized string similar to Thrist.
-        /// </summary>
-        internal static string ThristStrongConditionTitle {
+        internal static string SlotAux {
             get {
-                return ResourceManager.GetString("ThristStrongConditionTitle", resourceCulture);
+                return ResourceManager.GetString("SlotAux", resourceCulture);
             }
         }
         
-        /// <summary>
-        ///   Looks up a localized string similar to Ubnormal.
-        /// </summary>
-        internal static string UbnormalAttributeValueTitle {
+        internal static string TakePropInInventoryButtonTitle {
             get {
-                return ResourceManager.GetString("UbnormalAttributeValueTitle", resourceCulture);
+                return ResourceManager.GetString("TakePropInInventoryButtonTitle", resourceCulture);
             }
         }
         
-        /// <summary>
-        ///   Looks up a localized string similar to Use.
-        /// </summary>
-        internal static string UseCommandButtonTitle {
+        internal static string PutPropInContainerButtonTitleTemplate {
             get {
-                return ResourceManager.GetString("UseCommandButtonTitle", resourceCulture);
+                return ResourceManager.GetString("PutPropInContainerButtonTitleTemplate", resourceCulture);
             }
         }
         
-        /// <summary>
-        ///   Looks up a localized string similar to Mortal wound!.
-        /// </summary>
-        internal static string WoundCriticalConditionTitle {
+        internal static string FakeContainerName {
             get {
-                return ResourceManager.GetString("WoundCriticalConditionTitle", resourceCulture);
+                return ResourceManager.GetString("FakeContainerName", resourceCulture);
             }
         }
         
-        /// <summary>
-        ///   Looks up a localized string similar to Lesser wound.
-        /// </summary>
-        internal static string WoundLesserConditionTitle {
+        internal static string MainMenuButtonTitle {
             get {
-                return ResourceManager.GetString("WoundLesserConditionTitle", resourceCulture);
+                return ResourceManager.GetString("MainMenuButtonTitle", resourceCulture);
             }
         }
         
-        /// <summary>
-        ///   Looks up a localized string similar to Strong wound.
-        /// </summary>
-        internal static string WoundStrongConditionTitle {
+        internal static string NextScreenButtonTitle {
             get {
-                return ResourceManager.GetString("WoundStrongConditionTitle", resourceCulture);
+                return ResourceManager.GetString("NextScreenButtonTitle", resourceCulture);
```

Comment:
Looks like local R# cleaned up autogenerated file. You should configure R# and revert this changes as unexpected.

Reply:
It's not R# cleaned up but it's was rewritten by Rider when I added items to resources.